shared/src/bundles/users/types/user-update-profile-request-dto.type.ts

```diff
@@ -0,0 +1,16 @@
+import { type ValueOf } from '../../../types/types.js';
+import { type Gender } from '../users.js';
+
+type UserUpdateProfileRequestDto = {
+    id: number | null;
+    fullName?: string | null;
+    username?: string | null;
+    avatarUrl?: string | null;
+    dateOfBirth?: string | null;
+    weight?: number | null | '';
+    height?: number | null | '';
+    gender?: ValueOf<typeof Gender>;
+    [key: string]: string | number | null;
+};
```

Comment:
id is redundant, seems like every field should be required, avatar will be uploaded with separate request, index in this type is also redundant